marklogic-rdf4j/src/main/java/com/marklogic/semantics/rdf4j/MarkLogicRepository.java

```diff
@@ -322,17 +310,17 @@ public synchronized MarkLogicClient getMarkLogicClient() {
     }
 
     /**
-     * sets MarkLogicClient used by this repository
+     * Sets MarkLogicClient used by this repository.
      *
-     * @param client
+     * @param client the MarkLogicClient to be used
```

Comment:
I know it's hard to say something useful sometimes, esp. for a setter, but if the sentence is meaningless probably you shouldn't bother.  I'd say here it's important to know that the MarkLogicClient mediates all interaction with the database.